src/components/token/TokenAutoCompleteField.js

```diff
@@ -4,9 +4,14 @@ import Loading from '../Loading';
 import tokensApi from '../../api/tokensApi';
 import { debounce, get } from 'lodash';
 import { constants as hathorLibConstants } from '@hathor/wallet-lib';
+import { connect } from "react-redux";
 
 const DEBOUNCE_SEARCH_TIME = 200; // ms
 
+const mapStateToProps = (state) => {
+  return { serverInfo: state.serverInfo };
```

Comment:
Why do you need this? Is it in use?

Reply:
Nice catch, the placeholder in this file used "Hathor (HTR) ..." so I had to change it but I think I skipped this line.